src/core/Microsoft.PowerToys.Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -32,6 +34,18 @@ public GeneralViewModel()
             try
             {
                 GeneralSettingsConfigs = SettingsUtils.GetSettings<GeneralSettings>(string.Empty);
+
+                if (Helper.CompareVersions(GeneralSettingsConfigs.PowertoysVersion, Helper.GetProductVersion()) < 0)
+                {
+                    // Update settings
+                    GeneralSettingsConfigs.PowertoysVersion = Helper.GetProductVersion();

```

Comment:
currently, this is also hardcoded in the models, are you going to adjust that too?

Reply:
Do you mean `GeneralSettings.cs`? I changed the verision there too.

Reply:
You already did it in GeneralSettings.cs. But other models have this value hardcoded during initialization. Like this one for example:
![image](https://user-images.githubusercontent.com/58791731/81013401-e7a18f80-8e0f-11ea-9262-59c0001296b2.png) the value is initialized to an empty string.

Reply:
Why do we keep the version number in multiple files? Is it the same version number, or a module specific version?

Reply:
In my `settings.json` files for FancyZones and ShortcutGuide, it says 1.0, which is not the PowerToys version, and its also not in the same version number format.

Reply:
it is the powertoys version. that 1.0 value was hardcoded in SettingsV2 branch